pkg/workflows/dontime/provider.go

```diff
@@ -0,0 +1,45 @@
+package dontime
+
+import (
+	"context"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+)
+
+// Provider wraps an existing OCR3 plugin provider (from the relayer)
+// and overrides the plugin factory and contract transmitter for DonTime.
+type Provider struct {
```

Comment:
why?

Reply:
It's needed for [initializing the plugin](https://github.com/smartcontractkit/chainlink/pull/18355/files#diff-ee42cd8d3c2284e2aa914b4f7797b308ef4bf9f3e73a5692da84685f3bf9f542R673-R678)